.gitignore

```diff
@@ -10,3 +10,15 @@ local.cfg
 .idea/
 node_modules
 .DS_Store
+/service/static/css/bootstrap-jisc.min.css
+/service/static/css/muk.APCRenderer.min.css
+/service/static/css/muk.funder.min.css
+/service/static/css/muk.institution.min.css
+/service/static/css/muk.publisher.min.css
+/service/static/css/muk.search.min.css
+/service/static/css/service.min.css
+/service/static/js/muk.funder.min.js
+/service/static/js/muk.institution.min.js
+/service/static/js/muk.min.js
+/service/static/js/muk.publisher.min.js
+/service/static/js/muk.search.min.js
```

Comment:
Minified files should only be excluded from Git if, as part of the deployment process (from Dev to Test, AND from Test to Live), the minified files will be auto-generated. 

If ALL our css and js assets are being minified, then a couple of exclusion patterns (rather than specifying files  individually) might be better in .gitignore.